pages/docs/plugins/grouparoo-postgres.mdx

```diff
@@ -0,0 +1,205 @@
+- `password`: Password is the user's password. You may also want to delete this or set it to `undefined` if working with localhost.
+- `ssl`: (**required**, default: `false`) Enforce SSL connections only. Grouparoo will use SSL optionally if supported by the server. If using SSL, you can also pass the following options:
+  - `ssl_cert`
+  - `ssl_key`
+  - `ssl_ca`
+
+<ValidateAndApplyConfig />
+
+## Creating a Postgres Source
+
+A Source is how you import data into Grouparoo. For columnar sources, like Postgres, there are two _types_ of sources:
+
+- A _Table Source_ targets specific columns within a single table and can perform aggregation methods on that column.
+- A _Query Source_ provides the ability to write custom SQL code to extract data from one or more tables and import the result into Grouparoo.
+
+You can generate each type of source using the `generate` command. Each requires that you specify a `parent`, which should match the `id` of the App you created.
+
+    $ grouparoo generate postgres:table:source users --parent my_data_warehouse
+    $ grouparoo generate postgres:query:source purchases --parent my_data_warehouse
+
+Each generator has additional options for speeding up the config process. You can use the `--describe` flag to learn more about each command. For example:
+
+    $ grouparoo generate postgres:table:source --describe
+
+Some generators have a way to batch multiple generated config files together. [Read more here](/docs/cli/config#batch-generation).
+
+All Sources have a common set of options, [which you can read about here](/docs/config/sources/community#configuring-your-source). Type-specific options are listed below.
+
+### Table Source
+
+There are two objects in the Table Source config file that are unique to this type: `options` and `mappings`. More on these and other related items below.
+
+#### Table Source Options
+
+There is only one option:
+
+- `table`: (**required**) Name of the table in the Postgres database.
+
+#### Table Source Mappings
+
+Mapping is a section of key-value pairs which represents how Grouparoo should match source columns to properties in Grouparoo.
+
+For example, let's say your database has a column named `email` and `first_name`, while the Property `id` values in Grouparoo are `emailAddress` and `firstName`. Your mapping would look like this:
+
+```js
+mapping: {
+  email: "emailAddress",
+  first_name: "firstName"
+}
+```
+
+#### Table Source Schedule
+
+A schedule is attached to every Source config file by default, commented out. If you want to import data from the source on a schedule (which is the typical behavior), then you should remove the comments and apply the necessary values.
+
+[You can read more about the common options here](/docs/config/sources/community#adding-a-schedule). The Postgres-specific options (those in the `options` object) are:
+
+- `column`: (**required**) The name of the column to use as the high water mark. This is a date column.
```

Comment:
We should talk more about what a high water mark is:
* Increasing over time (updated_at or user_id)
* we will only import things that are newer than last time